api/paymentmethods/response.php

```diff
@@ -73,6 +73,26 @@ public function __construct(TransactionResponse $response = null)
         
     }
 
+    /**
+     * Get code required for payment
+     *
+     * @param string $configCode
+     *
+     * @return string
+     */
+    protected function getPaymentCode(string $configCode): string
+    {
+        $mappings = [
+            'capayable' => 'in3'
+        ];
+
+        if(isset($mappings[$configCode])) {
+            return $mappings[$configCode];
+        }
+
+        return $configCode;
```

Comment:
why not do a simple if statement? this approach is used usually when you have multiple associations

Reply:
You're right that it can be an efficient solution for handling a single case. However, the approach I used was intentionally designed with scalability in mind. It ensures that if and when there's a need to handle multiple associations, the code is structured and ready to accommodate them with minimal changes.

Reply:
so you are not a adept of YAGN

Reply:
I was thinking more in Scalability Perspective, but you are right it will not be used